src/tests/test_magic.py

```diff
@@ -1925,6 +1925,7 @@ def test_whitespaces_linebreaks_near_first_token(ip, query):
 
     ip.user_global_ns["query"] = query
     out = ip.run_cell("%sql {{query}}").result
+    print(out)
```

Comment:
please check your PRs before requesting a review, this print statement looks like it was a debugging thing

Reply:
Sorry about that! 😞